<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1055</width>
    <height>437</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Task 1</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1031</width>
      <height>201</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="struct5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>1000</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Структура 5: </string>
     </property>
    </widget>
    <widget class="QLabel" name="struct2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>1000</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Структура 2: </string>
     </property>
    </widget>
    <widget class="QLabel" name="struct1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>1000</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Структура 1: </string>
     </property>
    </widget>
    <widget class="QLabel" name="struct3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>1000</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Структура 3: </string>
     </property>
    </widget>
    <widget class="QLabel" name="struct4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>1000</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Структура 4: </string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>1031</width>
      <height>171</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Имя</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="nameInput">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>10</y>
       <width>111</width>
       <height>26</height>
      </rect>
     </property>
     <property name="maxLength">
      <number>19</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Возраст</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="ageInput">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>40</y>
       <width>111</width>
       <height>27</height>
      </rect>
     </property>
     <property name="minimum">
      <number>18</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Рост</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="heightInput">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>70</y>
       <width>111</width>
       <height>27</height>
      </rect>
     </property>
     <property name="minimum">
      <double>1.500000000000000</double>
     </property>
     <property name="maximum">
      <double>2.500000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Пол (m / f)</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="genderInput">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>100</y>
       <width>111</width>
       <height>26</height>
      </rect>
     </property>
     <property name="maxLength">
      <number>1</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Активен</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="isActiveYES">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>130</y>
       <width>111</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Да</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="isActiveNO">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>130</y>
       <width>111</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Нет</string>
     </property>
    </widget>
    <widget class="QFrame" name="frame_3">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>10</y>
       <width>441</width>
       <height>131</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>251</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Кол-ов золотых медалей</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="goldInput">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>10</y>
        <width>81</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>301</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Кол-ов серебрянных медалей</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="silverInput">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>50</y>
        <width>81</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>301</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Кол-ов бронзовых медалей</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="bronzeInput">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>90</y>
        <width>81</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="addButton">
     <property name="geometry">
      <rect>
       <x>750</x>
       <y>10</y>
       <width>261</width>
       <height>131</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Добавить</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1055</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
